src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -1,43 +1,139 @@
 package cromwell.engine
 
-import akka.actor.Actor
+import akka.actor.{ActorRef, Props, Actor}
 import akka.actor.SupervisorStrategy.Stop
+import akka.event.LoggingReceive
+import cromwell.binding.{FullyQualifiedName, Workflow, WdlBinding, WdlValue}
 
 /**
  * This model assumes workflow actors are transient, created once per workflow instance.
  */
 object WorkflowActor {
 
-  // This is hand-waving over the actual engine-level representation of a workflow.
-  type WorkflowModel = String
+  case class Construct(binding: WdlBinding, inputs: Map[String, WdlValue])
```

Comment:
All this goes away per our dicussion this afternoon, the WorkflowActor will be delivered to the "prime mover" in a Startable state.  But yes whatever key values represent FQNs should use the `FullyQualifiedName` type.